docs/server-backup-version-comparison.md

```diff
@@ -36,6 +34,8 @@ example, overrides the parameter and the backup proceeds.
 $ xtrabackup --backup --no-server-version-check --target-dir=$mysql/backup1
 ```
 
+Overriding the `--no-server-version-check` parameter allows taking backups using a Percona XtraBackup version that is equal to or greater than your source server major version but up to the next LTS release. For example, if you use Percona XtraBackup 8.1.x, you can back up the source system from 8.1.x to 8.4.xx.
```

Comment:
remove the "but" 

major version of your source server up to the last release in the LTS series. 

Maybe?

Reply:
Updated